tests/conftest.py

```diff
@@ -10,32 +10,47 @@
 from pySPM.SPM import SPM_image
 from pySPM.Bruker import Bruker
 
-from topostats.filters import (extract_img_name, extract_channel, extract_pixel_to_nm_scaling, extract_pixels,
-                               align_rows, remove_x_y_tilt, average_background)
-from topostats.find_grains import (gaussian_filter, tidy_border, remove_objects, label_regions,
-                                   minimum_grain_size_pixels, colour_regions, region_properties)
+from topostats.filters import (
+    extract_img_name,
+    extract_channel,
+    extract_pixel_to_nm_scaling,
+    extract_pixels,
+    align_rows,
+    remove_x_y_tilt,
+    average_background,
+)
+from topostats.find_grains import (
+    gaussian_filter,
+    tidy_border,
+    remove_objects,
+    label_regions,
+    calc_minimum_grain_size,
+    colour_regions,
+    region_properties,
+)
+from topostats.grainstats import GrainStats
 from topostats.io import load_scan, read_yaml
 from topostats.utils import get_mask, get_threshold
 
 BASE_DIR = Path.cwd()
-RESOURCES = BASE_DIR / 'tests' / 'resources'
+RESOURCES = BASE_DIR / "tests" / "resources"
 
 RNG = np.random.default_rng(seed=1000)
 SMALL_ARRAY_SIZE = (10, 10)
 THRESHOLD = 0.5
-CHANNEL = 'Height'
+CHANNEL = "Height"
```

Comment:
I'm a bit lazy and use single quotes to save having to hold down `Shift` lots when typing but as with above [Black](https://black.readthedocs.io/en/stable/) has been applied on saving and has changed most if not all single to double quotes.

I think to be consistent it would be useful to have everyone using the same formatter (e.g. Black) to apply formatting on saving. I have a hook set in Emacs that rungs Black and for VScode there are similar configurations as described [here](https://github.com/microsoft/vscode-black-formatter).